config/validation.js

```diff
@@ -0,0 +1,34 @@
+import Joi from "joi";
+
+export const person_schema = Joi.object().keys({
+  names: Joi.string().min(3).max(40).required(),
+  email: Joi.string().email({
+    minDomainSegments: 2,
+    tlds: { allow: ["com", "net"] },
```

Comment:
why did you limit the domain don't give the limitation of domains, it must accept all kinds of emails, such as @rw.ac.gov this is valid. email, so change that